Source/SpacePort/ObjectBuilder/ParkingSpaceObjectBuilder.cs

```diff
@@ -0,0 +1,60 @@
+﻿//using System;
+//using System.Collections.Generic;
+//using System.Linq;
+//using System.Text;
+
+//namespace SpacePort
+//{
+//    public interface IAddParkingSpaceComponents
+//    {
+//        IAddParkingSpaceComponents AddSpaceShip(SpaceShip spaceShip);
+//        IAddParkingSpaceComponents AddOccupied(bool occupied);
+
+//        ParkingSpace BuildParkingSpace();
+//    }
+
+//    public class ParkingSpaceObjectBuilder : IAddParkingSpaceComponents
+//    {
+//        private int id;
+//        private SpaceShip spaceShip;
+//        private bool occupied;
+
+//        public ParkingSpaceObjectBuilder(int id)
+//        {
+//            this.id = id;
+//        }
+
+//        public static IAddParkingSpaceComponents AddID(int id) => new ParkingSpaceObjectBuilder(id);
+
+//        public IAddParkingSpaceComponents AddSpaceShip(SpaceShip spaceShip)
+//        {
+//            this.spaceShip = spaceShip;
+//            return this;
+//        }
+
+//        public IAddParkingSpaceComponents AddOccupied(bool occupied)
+//        {
+//            this.occupied = occupied;
+//            return this;
+//        }
+
+//        public ParkingSpace BuildParkingSpace()
+//        {
+//            return new ParkingSpace(this.id, this.spaceShip, this.occupied);
+//        }
+
+//        public static ParkingSpace BuildFromDb(int id)
+//        {
+//            SpaceParkContext context = new SpaceParkContext();
+//            ParkingSpaceModel model = ParkingSpaceModel.CreateModelFromDb(id).Result;
+//            SpaceShip spaceShip = SpaceShipObjectBuilder.BuildFromDb(context.SpaceShip.Where(o => o.SpaceShipID == id).Select(p => p.Owner.PersonID).First());
+//            return ParkingSpaceObjectBuilder
+//                   .AddID(model.ParkingSpaceID)
+//                   .AddSpaceShip(spaceShip)
+//                   .AddOccupied(model.Occupied)
+//                   .BuildParkingSpace();
+                
+                
+//        }
+//    }
+//}
```

Comment:
Delete all this green-code

Reply:
Det var tanken i slutändan. Tyvärr så gjorde nog vår planering att vi gick lite i stöpet och det resulterade i att en hel del kod vi tänkt återimplementera aldrig blev det. Det finns en hel del i här som vi hade velat använda oss av men så hann vi inte. Men vi lär oss =)